repos/system_upgrade/common/libraries/rpms.py

```diff
@@ -1,7 +1,38 @@
 from leapp.libraries import stdlib
+from leapp.libraries.common.config.version import get_source_major_version
 from leapp.models import InstalledRPM
 
 
+class LeappComponents(object):
+    FRAMEWORK = 'framework'
+    REPOSITORY = 'repository'
+    COCKPIT = 'cockpit'
+    TOOLS = 'tools'
+
+
+LEAPP_PACKAGES_MAP = {
+        LeappComponents.FRAMEWORK: {'7': {'pkgs': ['leapp', 'python2-leapp'],
+                                          'deps': ['leapp-deps']},
+                                    '8': {'pkgs': ['leapp', 'python3-leapp'],
+                                          'deps': ['leapp-deps']}
+                                    },
+        LeappComponents.REPOSITORY: {'7': {'pkgs': ['leapp-upgrade-el7toel8'],
+                                           'deps': ['leapp-upgrade-el7toel8-deps']},
+                                     '8': {'pkgs': ['leapp-upgrade-el8toel9'],
+                                           'deps': ['leapp-upgrade-el8toel9-deps']}
+                                     },
+        LeappComponents.COCKPIT: {'7': {'pkgs': ['cockpit-leapp']},
+                                  '8': {'pkgs': ['cockpit-leapp']}
+                                  },
+        LeappComponents.TOOLS: {'7': {'pkgs': ['snactor']},
+                                '8': {'pkgs': ['snactor']}
+                                }
+        }
```

Comment:
I do not like this to be public. I prefer to keep this private (part of the function or use `_` prefix. There is no code outside of this library that should access this directly. Exposing more stuff than needed creates for us bigger maintainance burden.

Reply:
I can call it `_LEAPP_PACKAGES_MAP`, no problem.